azure.sh

```diff
@@ -117,6 +117,98 @@ function cleanup_az_storage_accounts {
     done
 }
 
+function cleanup_az_image_galleries {
+    IFS=$'\n'
```

Comment:
I think that the function should reset IFS to the original value when it returns.

Reply:
Makes sense! Done.